package.json

```diff
@@ -50,7 +50,7 @@
     "babel-jest": "^29.5.0",
     "cypress": "^12.10.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^29.5.0",
+    "jest": "^29.7.0",
```

Comment:
This change seems unrelated

Reply:
While running the tests I noticed that new version for jest was out, so I added it. Should I make a separate PR or is it ok to not update the version?

Reply:
That's fine, but it should be it's own commit, at least.